src/foam/dao/EasyDAO.js

```diff
@@ -521,6 +529,16 @@ return delegate;
       name: 'lastModifiedByAware',
       class: 'Boolean',
       value: false
+    },
+    {
+      name: 'fixedSizeDAO',
```

Comment:
Alternative names: `fixedSize: true` and `size: 10000`. The other properties don't include "DAO" in the name so it would be nice to stay consistent.